extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/model/MultiValuedTreeMap.java

```diff
@@ -181,21 +181,19 @@ public boolean equalsIgnoreValueOrder(MultiValuedTreeMap<Key, Value> vmap) {
 
     public String toString() {
         StringBuilder result = new StringBuilder();
+        result.append("[");
         String delim = ",";
         for (Object name : keySet()) {
             for (Object value : get(name)) {
                 result.append(delim);
-                if (name == null) {
-                    result.append("null"); //$NON-NLS-1$
-                } else {
-                    result.append(name.toString());
-                }
+                result.append(name != null ? name : "null");
```

Comment:
Isn't easier to just do `result.append(name)`?

Reply:
Actually, better do it like: https://github.com/quarkusio/quarkus/blob/4278eb85b6e202840679dbf2e7c6400301b4f21f/independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/util/MultivaluedTreeMap.java#L111

Reply:
Thanks for your review, could you please review it again? I refactored except one subject. If i do `result.append(name)` , StringBuilder throwing `NullPointerException` when  doing `str.length()`